src/app/components/submit-request-page/submit-request-page.component.ts

```diff
@@ -31,7 +31,7 @@ export class SubmitRequestPageComponent implements OnInit {
   title: string;
 
   //Format of the request
-  selFormat: string;
+  selFormat: string = "Code";
```

Comment:
In so many other locations, selFormat has been changed to be structured as an array. I think we should be consistent with how we handle our formats.